source/docs/upgrade-to-v2.md

```
@@ -0,0 +1,88 @@
+---
+title: Upgrading to v2
+description: Learn what changed in Harbor v2 and how to migrate your existing workflows.
+extends: _layouts.documentation
+section: content
+---
+# Upgrading to v2 {#upgrading-to-v2}
+
+### [Overview](#overview) {#overview}
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)